tests/compile-fail/intrinsics/copy_null.rs

```diff
@@ -9,5 +9,7 @@ fn main() {
     let mut data = [0u16; 4];
     let ptr = &mut data[0] as *mut u16;
     // Even copying 0 elements from NULL should error.
-    unsafe { copy_nonoverlapping(std::ptr::null(), ptr, 0); } //~ ERROR: memory access failed: null pointer is not a valid pointer
+    unsafe {
+        copy_nonoverlapping(std::ptr::null(), ptr, 0);
+    } //~^ ERROR: memory access failed: null pointer is not a valid pointer
```

Comment:
IMO for cases like this (of which we have plenty) it'd be better to keep the comment in the same line as the function call, rather than have it after the unsafe block.

Reply:
So, I did this based on line length. Notice for example *tests/compile-fail/box-cell-alias.rs* which has a shorter function call and shorter comment, the change is like this:

```diff
-    unsafe { (*ptr).set(20); } //~ ERROR does not exist in the borrow stack
+    unsafe {
+        (*ptr).set(20); //~ ERROR does not exist in the borrow stack
+    }
```

Basically if the comment fit without exceeding ~80 columns, I put it on the same line, otherwise the next line. It's clear to compiletest from the `//~` vs `//~^` whether the expected error is talking about the current line vs previous line.

The alternatives I see are the following, but I find both less good than what the PR is currently doing:

```rust
    // very long line
    unsafe {
        copy_nonoverlapping(std::ptr::null(), ptr, 0); //~ ERROR: memory access failed: null pointer is not a valid pointer
    }
```

```rust
    // more vertical space for no real benefit, I think
    unsafe {
        copy_nonoverlapping(std::ptr::null(), ptr, 0);
        //~^ ERROR: memory access failed: null pointer is not a valid pointer
    }
```

Reply:
I find both better than what the PR is doing. :shrug: 
Having the comment after the `}` is rather odd IMO, it goes against the structure of the code.

Reply:
Okay! Will update.